streaming_main.py

```diff
@@ -0,0 +1,553 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021, David Tuin <mrdavidot@gmail.com>
```

Comment:
Not sure what to do with the copyright here since I did copy a lot?

Reply:
I think you should keep the original copyright and add your own also.